### PR TITLE
`ViewNavigationWidget` and `NavigationTabsWidget` are added

### PR DESCRIPTION
**Pull request by DRITE**
See #702

### NavigationTabsWidget
The purpose of the widget is to display navigation tabs depends on position in view's metadata. In other words this widget should be included in widgets stream.

### ViewNavigationWidget
The widget displays the same component as `NavigationTabsWidget`.
But this widget should be used in case need to exclude navigation  from widgets stream.For instance, client could decide to fix it on top of the screen.

### CODE REVIEW

File: src/components/ui/NavigationTabs/NavigationTabs.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+function NavigationTabs({ navigationLevel }: NavigationTabsProps) {
+    const tabs = useViewTabs(navigationLevel)
+    const handleChange = (key: string) => {
+        historyObj.push(key)
+    }
+
+    return (
+        <nav className={styles.container}>
+            <Tabs activeKey={tabs?.find(item => item.selected)?.url} tabBarGutter={24} size="large" onChange={handleChange}>
+                {tabs.map(item => (
```
Comment (Dergash) [inline src/components/ui/NavigationTabs/NavigationTabs.tsx:37]: No null check when there is one on previous line

File: src/components/widgets/NavigationTabsWidget/NavigationTabsWidget.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react'
+import NavigationTabs from '../../ui/NavigationTabs/NavigationTabs'
+import { NavigationWidgetMeta } from '../../../interfaces/widget'
+
+export interface NavigationTabsWidgetProps {
+    meta: NavigationWidgetMeta
```
Comment (Dergash) [inline src/components/widgets/NavigationTabsWidget/NavigationTabsWidget.tsx:23]: Why the entire meta object is a prop when only `navigationLevel` field is actually used?

File: src/hooks/useViewTabs.ts [hunk 3]
```diff
@@ -40,7 +40,7 @@ function mapStateToProps(store: Store): UseViewTabsState {
  * @param depth 1 for top level navigation; 2, 3, 4 for SecondLevelMenu, ThirdLevelMenu and FourthLevelMenu
  * @category Hooks
  */
-export function useViewTabs(depth: NavigationLevel) {
+export function useViewTabs(depth: NavigationLevel | number) {
```
Comment (Dergash) [inline src/hooks/useViewTabs.ts:43]: There is no point in this union: the resulting type will be extended to `number`. If all numbers now supported, `NavigationLevel` should be removed

**Review by Dergash — CHANGES_REQUESTED** [review]

File: src/components/ui/NavigationTabs/NavigationTabs.tsx (hunk 1 above)
Comment (DRITE) [reply src/components/ui/NavigationTabs/NavigationTabs.tsx]: fixed

File: src/hooks/useViewTabs.ts (hunk 3 above)
Comment (DRITE) [reply src/hooks/useViewTabs.ts]: removed

**Review by Dergash — APPROVED** [review]

**Pull request merged**